File: C++/خر در چمن فراوونه!!.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b, l, k{0}, sum{0};
    cin >> a >> b >> l;
    while (k < l)
    {
        sum += a;
        k++;
        if (k == l)
        {
            break;
        }
        sum += b;
        k++;
    }
    cout << sum;
}

File: Python/خر در چمن فراوونه!!.py
a, b, l = input().split()
a = int(a)
b = int(b)
l = int(l)
k = 0
sum = 0
while k < l:
    sum += a
    k += 1
    if k == l:
        break
    sum += b
    k += 1
print(sum)
